packages/syntax-core/src/Checkbox/Checkbox.stories.tsx

```diff
@@ -49,7 +49,14 @@ const CheckboxInteractive = ({
   const handleChange = () => {
     setIsChecked(!isChecked);
   };
-  return <Checkbox checked={isChecked} onChange={handleChange} label={label} />;
+  return (
+    <Checkbox
+      ariaValues={{ "aria-describedby": "checkboxes" }}
```

Comment:
@somethiiing does the `checkboxes` id actually exist on this page? If not, could we add it so we can test the behavior?